### PR TITLE
Add N-of-1 medium article to MDS blog

### PR DESCRIPTION
**Pull request by meretelutz**
Should be good to go! Thanks for wanting to feature my article!

### CODE REVIEW

File: _posts/2024-08-15-n-of-1-fitness-data.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# N-of-1 Trials and Analyzing Your Own Fitness Data
+
+## Do I really sleep worse after drinking alcohol?
+
+### By [Merete Lutz](https://meretelutz.github.io/)
```
Comment (zmx721) [inline _posts/2024-08-15-n-of-1-fitness-data.md]: Suggested change for the format:

---

layout: post
title: "N-of-1 Trials and Analyzing Your Own Fitness Data"
subtitle: by [Merete Lutz](https://meretelutz.github.io/)

---

## Do I really sleep worse after drinking alcohol?

**Review by phchen5 — DISMISSED** [review]
lgtm

File: _posts/2024-08-15-n-of-1-fitness-data.md (hunk 1 above)
Comment (meretelutz) [reply _posts/2024-08-15-n-of-1-fitness-data.md]: Sounds good! I've made the change.

**Comment by zmx721** [comment]
Sanity check by Katie

**Pull request merged**